docs/source/roadmap.rst

```diff
@@ -65,6 +60,14 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v3.2
+
+* **RHEL/CentOS:** Support CentOS 8/RHEL 8 with Python 3.6 and MongoDB 4.0.
+* **Ubuntu:** Stop producing Ubuntu 14.04 packages.
```

Comment:
```suggestion
* **Ubuntu:** Drop support for Ubuntu 14.04.
```
Thoughts?

Reply:
We dropped support in `v3.1`, see `Done in v3.1` section.

Reply:
We also dropped all testing of the Ubuntu 14.04 packages. Ideally, every packaged environment is also tested and officially supported, but that's not always the case, and it wasn't the case here.

I don't know how relevant that will be to any/many users though.

Reply:
To recap. It's possible to stop supporting the OS/distro, while still shipping the packages. This means no support, help, bugfixes provided as well as no issues accepted related to that distro/installation.

We added U18 and dropped U14 support in `v3.1`. By shipping the U14 packages one more release after `v3.1` we just gave some time to someone who was still relying on that but couldn't switch to other supported OS/distro.

And so as result Roadmap resulted in the following:
```
## Done in v3.2
**Ubuntu:** Stop producing Ubuntu 14.04 packages.

## Done in v3.1
**Ubuntu:** Drop Ubuntu 14.04 support. Packages are still available for a limited time.
**Ubuntu:** GA Support Ubuntu 18.04, with Python 3.6
```

@nmaludy I hope that makes more sense now.

Reply:
Awesome!